installed react test library and refactored sample tests

jleiandy opened the pull request:
installed react test library and refactored sample tests.
deleted enzyme related files and uninstalled enzyme node modules.

jleiandy committed:
installed react test library and refactored sample tests

cschucode commented on client/components/JestReact.test.jsx:
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen} from '@testing-library/react';
```
Missing whitespace after screen. Were these files linted? I can add that task to circleCI as well.

jleiandy replied:
Good eye, I did not see the lint on review because it's hard for me to see the red. I will get into the habit of running the lint command before each commit.

cschucode commented on client/components/JestReact.test.jsx:
```diff
@@ -0,0 +1,29 @@
+import userEvent from '@testing-library/user-event';
+import App from './JestComponent';
+import '@testing-library/jest-dom';
+
```
I would also group your imports following a consistent pattern with a line between the different types in this order 
1. builtin
2. external
3. internal
see https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m

jleiandy replied:
I think I organized it correctly. Not entirely sure what falls under builtin. Please check.

cschucode replied:
Built-in would be node libraries like 'fs' or 'path' that we don't need to explicitly install via our package.json. External would be things like react and testing-library etc, so I think those things wouldn't need the extra space on line 2, and the App import looks correct with one line space between it and the external imports. 👍

jleiandy committed:
Fixed linting
Fixed linting in import statements in JestReact file and also
fixed linting in Jest.config file

jleiandy committed:
Deleted extra line from import statements.
